src/components/Projects/Section/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import SectionList from '../SectionList';
+
+import './index.scss';
+
+const Section = ({ list }) => {
+  return (
+    <div className="section">
+      <div className="section__title">Projects</div>
```

Comment:
**WARNING**: The same styles and DOM structure are being repeated

**SOLUTION**: Create reusable component to reduce bad practice